client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/fallback/ShuffleFallbackPolicyRunner.java

```diff
@@ -32,7 +32,7 @@ public class ShuffleFallbackPolicyRunner {
   private static final List<ShuffleFallbackPolicy> FALLBACK_POLICIES =
       ShuffleFallbackPolicyFactory.getShuffleFallbackPolicies();
 
-  public static boolean applyFallbackPolicies(
+  public static Optional<ShuffleFallbackPolicy> applyFallbackPolicies(
```

Comment:
I think this function name can be changed into `getActiveFallbackPolicy` to match its logic.